content/community/gsoc-2022.es.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: XSF y Google Summer of Code 2022
+---
+
+XSF y GSoC 2022 Logo](/images/logos/GSoC_2022_Logo.png)
+
+La XMPP Standards Foundation (XSF) tiene previsto participar como organización anfitriona en el [Google Summer of Code 2022](https://summerofcode.withgoogle.com/).
+Si estás interesado en participar como estudiante (o simplemente recién llegado al código abierto), mentor o tienes un proyecto XMPP, ¡Ésta es la página por la que debes empezar!.
+ 
+Si te imaginas trabajando a tiempo parcial (o incluso a tiempo completo) en un proyecto de software de código abierto durante el verano, entonces considera unirte a un proyecto de tecnología de comunicación en tiempo real XMPP y ¡participa en el Google Summer of Code 2022!
+
+### XMPP - el Protocolo Extensible de Mensajería y Presencia
```

Comment:
can we keep the part in English?

XMPP - el Extensible Messaging and Presence Protocol

Reply:
Sure,but like I said in the MUC ... better if its all the way in english